src/common/messages.cpp

```diff
@@ -53,6 +53,39 @@ const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap()
     return FEE_MODES;
 }
 
+std::string FeeModeInfo(const std::pair<std::string, FeeEstimateMode>& mode)
+{
+    switch (mode.second) {
+        case FeeEstimateMode::UNSET:
+            return std::string("");
+        case FeeEstimateMode::ECONOMICAL:
+            return std::string("Economical estimates use a shorter time horizon, making them more\n"
+                   "responsive to short-term drops in the prevailing fee market. This mode\n"
+                   "potentially returns a lower fee rate estimate.\n");
+        case FeeEstimateMode::CONSERVATIVE:
+            return std::string("Conservative estimates use a longer time horizon, making them\n"
+                   "less responsive to short-term drops in the prevailing fee market. This mode\n"
+                   "potentially returns a higher fee rate estimate.\n");
+        default:
+            // no default case, so the compiler can warn about missing cases
+            assert(false);
+    }
+}
+
+std::string FeeModesDetail()
+{
+    std::string modes{"Available modes: "};
+    std::string info;
+    for (const auto& fee_mode : FeeModeMap()) {
+        modes += strprintf("\"%s\", ", fee_mode.first);
+        info += FeeModeInfo(fee_mode);
+    }
+    if (!FeeModeMap().empty()) {
+        modes = modes.substr(0, modes.size() - 2) + "\n";
```

Comment:
nitty nit:
If changing this file, could add a comment for clarity, although it's a small enough function for a reader to pick up on the meaning of the magic number 2.

```c++
modes = modes.substr(0, modes.size() - 2) + "\n"; // remove the ", " delimiter
```

Alternatively, could add a string to name the delimiter.  Something like:
 ```diff
 diff --git a/src/common/messages.cpp b/src/common/messages.cpp
index bf41d09bbd..76777dd8cd 100644
--- a/src/common/messages.cpp
+++ b/src/common/messages.cpp
@@ -76,12 +76,13 @@ std::string FeeModesDetail()
 {
     std::string modes{"Available modes: "};
     std::string info;
+   std::string delimiter{", "};
     for (const auto& fee_mode : FeeModeMap()) {
-        modes += strprintf("\"%s\", ", fee_mode.first);
+        modes += "\"" + fee_mode.first + "\"" + delimiter;
         info += FeeModeInfo(fee_mode);
     }
     if (!FeeModeMap().empty()) {
-        modes = modes.substr(0, modes.size() - 2) + "\n";
+        modes = modes.substr(0, modes.size() - delimiter.length()) + "\n";
     }
     return modes + info;
 }
```

Reply:
Taken.